src/shared/lib/formatEventRequest.ts

```
@@ -0,0 +1,19 @@
+import { BaseEvent } from '../types/baseEventType';
+
+export const formatEventRequest = (data: any): BaseEvent => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)